docs/reference/snapshot-restore/apis/create-snapshot-api.asciidoc

```diff
@@ -133,7 +133,7 @@ default. If `include_global_state` is `false`, the snapshot includes no feature
 states by default.
```

Comment:
I think we should explicitly state that an empty array results in default behavior here.

Many users may assume an empty array == no feature states.